README.md

```diff
@@ -1,26 +1,26 @@
 # System.Web.Bem
 
-System.Web.Bem - БЭМ-инфрастурктура для ASP.NET MVC.
+System.Web.Bem - is a BEM (Block-Element-Modifier) infrastructure for ASP.NET MVC projects.
 
-- [Быстрый старт](#Быстрый-старт)
-- [Подробнее](#Подробнее)
-    - [Специфика БЭМ-проектов](#Специфика-БЭМ-проектов)
-    - [Структура проекта](#Структура-проекта)
-    - [Сборка](#Сборка)
-    - [Серверная шаблонизация BEMHTML](#Серверная-шаблонизация-bemhtml)
-    - [Подключение библиотек блоков](#Подключение-библиотек-блоков)
-- [Публикации](#Публикации)
+- [Get Started](#get-started)
+- [Details](#details)
+    - [Specificity of BEM projects](#specificity-of-bem-projects)
+    - [Structure of project files](#structure-of-project-files)
+    - [Build the bundles](#build-the-bundles)
+    - [Server side BEMHTML templates](#server-side-bemhtml-templates)
+    - [External block libraries usage](#external-block-libraries-usage)
+- [Articles](#articles)
 
-## Быстрый старт
+## Get Started
 
-1. Убедитесь, что на компьютере, где будет *выполняться сборка* проекта, установлен [node.js](https://nodejs.org/en/) (при этом для *работы приложения* устанавливать node.js не обязательно).
+1. Make sure that [node.js](https://nodejs.org/en/) is installed on the computer that will assemble of the project (node.js on the production server is not required).
 
-1. Установите [пакет System.Web.Bem](https://www.nuget.org/packages/System.Web.Bem/) в свой проект ASP.NET MVC.
+1. Install the [System.Web.Bem package](https://www.nuget.org/packages/System.Web.Bem/) into your ASP.NET MVC project.
   ```bash
   PM> Install-Package System.Web.Bem -Pre
   ```
 
-1. Верните из метода контроллера экземпляр класса `BemhtmlResult`, передав ему в конструктор нужный bemjson.
+1. Return the instance of the `BemhtmlResult` class from controller's method. Send the essential bemjson to constructor of `BemhtmlResult`.
```

Comment:
-> from the controller's method and send the necessary bemjson to the `BemhtmlResult` constructor.